src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -14,26 +14,37 @@
 
 from __future__ import absolute_import
 from __future__ import print_function
+
+import urlparse
 from Queue import Queue
 from abc import abstractmethod, ABCMeta
 import hashlib
 from itertools import chain, islice
 import logging
 import os
+import io
 import urllib2
 from threading import Thread
 import tempfile
 import uuid
 import shutil
 import time
+import itertools
+import boto
+import codecs
+
 from unittest import skip
+from azure.storage import BlobService
 from toil.common import Config
 from toil.jobStores.abstractJobStore import (AbstractJobStore, NoSuchJobException,
                                              NoSuchFileException)
-from toil.jobStores.fileJobStore import FileJobStore
-from toil.test import ToilTest, needs_aws, needs_azure, needs_encryption
 
+from bd2k.util.objects import abstractstaticmethod, abstractclassmethod
+from toil.jobStores.fileJobStore import FileJobStore
+from toil.lib import encryption
+from toil.test import ToilTest, needs_aws, needs_azure, needs_encryption, make_tests
 logger = logging.getLogger(__name__)
+mpTestPartSize = 2**20 * 5
```

Comment:
Can we make this a class attribute of AJST and initialize right before makeImportExportTests?